.github/workflows/test.yml

```diff
@@ -11,16 +11,28 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-
+      - uses: actions/checkout@v3.1.0
+      - uses: technote-space/get-diff-action@v6.1.1
+        with:
+          PATTERNS: |
+            **/*.go
+            go.mod
+            go.sum
```

Comment:
I would also include any files in `testdata` folders, because they are used in the tests.

Reply:
e14ba5391b06e35ed5dff9eae8a6a07e29d593f2